ruff.toml

```diff
@@ -4,7 +4,7 @@ target-version = "py310"
 [lint]
 select = ["ALL"]
 
-ignore = []
+ignore = ["D203","D212","UP015"]
```

Comment:
What do you think about having a space after each comma?

Reply:
I'm not sure if ruff has some codestyle for this, but I usually don't put spaces for something like this. 
I'm fine with or without though so your choice.

Reply:
Please do, it's a standard codestyle as seen from the official TOML page: https://toml.io/en/

Example:
```toml
[database]
enabled = true
ports = [ 8000, 8001, 8002 ]
data = [ ["delta", "phi"], [3.14] ]
temp_targets = { cpu = 79.5, case = 72.0 }
```